7_days_to_die/egg-7-days-to-die.json

```
@@ -32,103 +33,171 @@
+            "default_value": "1",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": [
+                "required",
+                "boolean"
+            ]
+        },
+        {
+            "sort": 4,
+            "name": "Crossplay Support",
+            "description": "Enables\/Disables crossplay",
+            "env_variable": "CROSSPLAY_SUPPORT",
+            "default_value": "false",
             "user_viewable": true,
             "user_editable": false,
-            "rules": "required|string|max:20",
-            "sort": null,
-            "field_type": "text"
+            "rules": [
+                "required",
+                "string",
+                "in:true,false"
+            ]
         },
         {
+            "sort": 3,
+            "name": "Easy Anti-Cheat",
+            "description": "Enables\/Disables EasyAntiCheat",
+            "env_variable": "EAC",
+            "default_value": "true",
+            "user_viewable": true,
+            "user_editable": false,
+            "rules": [
+                "required",
+                "string",
+                "in:true,false"
+            ]
+        },
+        {
+            "sort": 2,
             "name": "Game Difficulty",
             "description": "0 - 5, 0=easiest, 5=hardest",
             "env_variable": "GAME_DIFFICULTY",
             "default_value": "2",
             "user_viewable": true,
             "user_editable": true,
-            "rules": "required|integer|between:0,5",
-            "sort": null,
-            "field_type": "text"
+            "rules": [
+                "required",
+                "integer",
+                "between:0,5"
+            ]
         },
         {
-            "name": "Source AppID",
-            "description": "This is the app id for 7dtd please no step on snek.",
-            "env_variable": "SRCDS_APPID",
-            "default_value": "294420",
-            "user_viewable": false,
-            "user_editable": false,
-            "rules": "required|string|max:20",
-            "sort": null,
-            "field_type": "text"
-        },
-        {
-            "name": "Auto Update",
-            "description": "This is to auto update the server on start.\r\n\r\nOptions are 0 or 1\r\nDefault is 1",
-            "env_variable": "AUTO_UPDATE",
-            "default_value": "1",
+            "sort": 5,
+            "name": "Ignore EOS Sanctions ",
+            "description": "Ignore EOS sanctions when allowing players to join",
+            "env_variable": "IGNORE_EOS_SANCTIONS",
+            "default_value": "false",
             "user_viewable": true,
-            "user_editable": true,
-            "rules": "required|boolean",
-            "sort": null,
-            "field_type": "text"
+            "user_editable": false,
+            "rules": [
+                "required",
+                "string",
+                "in:true,false"
+            ]
```

```suggestion
            "rules": [
                "required",
                "boolean"
             ]
```
https://laravel.com/docs/12.x/validation#rule-boolean